internal/cmd/controller/reconciler/gitrepo_controller.go

```diff
@@ -271,7 +271,7 @@ func purgeBundles(ctx context.Context, c client.Client, gitrepo types.Namespaced
 		}
 
 		err = purgeBundleDeployments(ctx, c, types.NamespacedName{Namespace: bundle.Namespace, Name: bundle.Name})
-		if err != nil {
+		if client.IgnoreNotFound(err) != nil {
 			return err
```

Comment:
Sorry, should've pointed out you can simplify this one too....

Basically `client.IgnoreNotFound` returns nil if err is a not found error, else it returns the error.

But this is definitely a nitpick.

Reply:
Makes sense, I could have thought of it TBH... On the other hand, I was having doubts as that line is already long enough, but... Done :)